### DISCUSSION TITLE
Chat area

### DISCUSSION THREAD

**Discussion by Cibiyanna26** [Cibiyanna26/EffiTrack#25] [General]
Lets push pass the limits

**Comment by Cibiyanna26** [comment]
Guys 📢

Lets have iteration from 8/1/24 to 26/1/24

We have to complete the work before 26th and will launch our mvp on 27th jan 🙌

**Reply by muruga21** [reply]
working on it

**Comment by Cibiyanna26** [comment]
We have released the V1.0

**Comment by Cibiyanna26** [comment]
I have created a new branch v1.1 for the next version create sub branch from them

**Comment by Cibiyanna26** [comment]
so let's make our next release within 10 days and with high quality without any bugs and It gonna rock ❤️‍🔥

**Comment by Cibiyanna26** [comment]
after the next release let's make it open source again

**Reply by muruga21** [reply]
y bro ?
